app/components/impagination-dataset.js

```diff
@@ -1 +1,116 @@
-export { default } from 'ember-impagination/components/impagination-dataset';
```

Comment:
Chris and I designed the addon this way so that we could use `ember-browserify`. We got it to work by moving a lot of logic into the `app` folder. 

Any suggestions about how to architect this correctly?